.env

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+export CODER_GITAUTH_0_TYPE=github
+export CODER_GITAUTH_0_CLIENT_ID=Iv1.1a90e650a1b10f44
+export CODER_GITAUTH_0_CLIENT_SECRET=724db3275e13fe30a21844287cbeac17956cba5f
```

Comment:
!

Reply:
;p these are just for local dev anyways, so not a biggie but still a mistake haha